omas/omas_plot.py

```diff
@@ -363,7 +363,7 @@ def geo_type_lookup(geometry_type, subsys, imas_version=None, reverse=False):
     """
 
     if imas_version is None:
-        imas_version = ODS().imas_version
+        imas_version = '99.99.99'  # Unrecognized
```

Comment:
@eldond please always use in your functions

```python
def bla(imas_version=default_imas_version):
   ...
```

Reply:
@orso82 YES! Thank you. This is what I was looking for. What's the best way to search OMAS documentation?

Reply:
well, it is online: https://gafusion.github.io/omas/code.html

Maybe the question for you is where you would have expected this information to be... we can then put it there ;)

Reply:
No, I just needed to realize it was available. Anyway, I think it should be fixed and ready to merge. Let me know if it needs anything else.